### PR TITLE
feat: ASC-00000 - create livestream

### PR DESCRIPTION
**Pull request by ptchayap**

### CODE REVIEW

File: src/components/PostTypeChoiceModal/PostTypeChoiceModal.tsx [hunk 1]
```diff
@@ -41,7 +45,13 @@ const PostTypeChoiceModal = () => {
...
     if (targetId && targetName && targetType) {
       const targetscreen =
-        type === 'post' ? 'CreatePost' : type === 'poll' ? 'CreatePoll' : null;
+        type === 'post'
+          ? 'CreatePost'
+          : type === 'poll'
+          ? 'CreatePoll'
+          : type === 'livestream'
+          ? 'CreateLivestream'
+          : null;
```
Comment (frankent) [inline src/components/PostTypeChoiceModal/PostTypeChoiceModal.tsx:54]: Optional, In my opinion if it is a dedicate function which handle as early return for each type it should easier to read

File: src/components/PostTypeChoiceModal/PostTypeChoiceModal.tsx [hunk 2]
```diff
@@ -119,6 +129,17 @@ const PostTypeChoiceModal = () => {
...
+          <TouchableOpacity
+            onPress={() => onChooseType('livestream')}
+            style={styles.modalRow}
+          >
+            <SvgXml
+              xml={livestreamIcon(theme.colors.base)}
+              width="28"
+              height="28"
+            />
+            <Text style={styles.postText}>Livestream</Text>
```
Comment (frankent) [inline src/components/PostTypeChoiceModal/PostTypeChoiceModal.tsx:141]: Please use i18n for all text (if it already setup, please check with @NaingAmity )

File: src/hooks/useImagePicker.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+  const openImageGallery = async () => {
+    await launchImageLibrary(options, (response) => {
+      if (response.didCancel) {
+        console.log('User cancelled image picker');
+      } else if (response.errorCode) {
+        console.log(
+          'ImagePicker Error: ',
+          response.errorCode + ', ' + response.errorMessage
+        );
```
Comment (frankent) [inline src/hooks/useImagePicker.ts:25]: if this code for debugging purposes, please don't forget to remove

File: src/screens/CreateLivestream/index.tsx [hunk 4]
```diff
@@ -0,0 +1,377 @@
...
+  const requestPermission = async () => {
+    try {
+      let granted = await PermissionsAndroid.request(
+        PermissionsAndroid.PERMISSIONS.CAMERA,
+        {
+          title: 'Camera Permission',
+          message: 'App needs access to your camera',
+          buttonNeutral: 'Ask Me Later',
+          buttonNegative: 'Cancel',
+          buttonPositive: 'OK',
```
Comment (frankent) [inline src/screens/CreateLivestream/index.tsx:61]: This file too, is it possible to use i18n ?

**Review by frankent — COMMENTED** [review]
Question krub

**Comment by NaingAmity** [comment]
LGTM !!

**Review by NaingAmity — APPROVED** [review]

**Pull request closed without merging**